testament/important_packages.nim

```diff
@@ -10,9 +10,9 @@ pkg "arraymancer", "nim c src/arraymancer.nim", "", true
 pkg "ast_pattern_matching", "nim c tests/test1.nim"
 pkg "blscurve", "", "", true
 pkg "bncurve", "nim c tests/tarith.nim", "", true
-pkg "c2nim"
+pkg "c2nim", "nim c testsuite/tester.nim"
 pkg "chronicles", "nim c -o:chr chronicles.nim"
-pkg "chronos", "nim c tests/teststream.nim"
+# pkg "chronos", "nim c tests/teststream.nim"
 pkg "cligen", "nim c -o:cligenn cligen.nim"
 pkg "compactdict", "nim c tests/test1.nim"
```

Comment:
Sucks that you need to do this, don't these package define a `test` task?

Reply:
They don't and this solution is fine for the time being, we can't test too many packages with this approach anyway.